test/evp_kdf_test.c

```diff
@@ -20,14 +20,18 @@
 static int test_kdf_tls1_prf(void)
 {
     int ret;
-    EVP_KDF_CTX *kctx;
+    EVP_KDF_CTX *kctx = NULL;
+    const EVP_KDF *kdf;
     unsigned char out[16];
     const unsigned char expected[sizeof(out)] = {
         0x8e, 0x4d, 0x93, 0x25, 0x30, 0xd7, 0x65, 0xa0,
         0xaa, 0xe9, 0x74, 0xc3, 0x04, 0x73, 0x5e, 0xcc
     };
 
-    ret = TEST_ptr(kctx = EVP_KDF_CTX_new_id(EVP_KDF_TLS1_PRF))
+    ret = TEST_ptr(kdf = EVP_get_kdfbyname(SN_tls1_prf))
+          && TEST_ptr(kctx = EVP_KDF_CTX_new(kdf))
+          && TEST_ptr_eq(EVP_KDF_CTX_kdf(kctx), kdf)
+          && TEST_str_eq(EVP_KDF_name(kdf), SN_tls1_prf)
```

Comment:
Code style nit: 4 space indent.

I know that this was the indentation of the original work, and obviously glossed over.  Would you mind adjusting all lines?